packages/manager/apps/cloud/client/app/deskaas/deskaas-get-console-access/index.js

```diff
@@ -0,0 +1,13 @@
+import angular from 'angular';
+import component from './deskaas-get-console-access.component';
+import routing from './deskaas-get-console-access.routing';
+
+const moduleName = 'managerApp';
```

Comment:
Shouldn't you define your own component instead ?

Reply:
I will have to rewrite the main module for this. Is it okay ?
The ticket was only about modal

Reply:
Oh got it....ignore the previous comment